api/handlers/service_plan.go

```diff
@@ -17,8 +17,9 @@ import (
 )
 
 const (
-	ServicePlansPath          = "/v3/service_plans"
-	ServicePlanVisivilityPath = "/v3/service_plans/{guid}/visibility"
+	ServicePlansPath             = "/v3/service_plans"
+	ServicePlanVisibilityPath    = "/v3/service_plans/{guid}/visibility"
+	ServicePlanVisibilityOrgPath = ServicePlanVisibilityPath + "/{org-guid}"
```

Comment:
In my opinion it would be more readable if the path is a plain single string:

```
ServicePlanVisibilityOrgPath = "/v3/service_plans/{guid}/visibility/{org-guid}"
```

Having it plain makes it easier to get what endpoints are implemented with a single glance. It also makes it easier to extract this information with a script